generate-gnostic/test/errors/x-extension-name-collision.errors

```diff
@@ -1,2 +1 @@
-Schema SampleCompanyTwoPrimitiveString and SampleCompanyOnePrimitiveString have the same 'id' field value.
-
+schema SampleCompanyTwoPrimitiveString and SampleCompanyOnePrimitiveString have the same 'id' field value
```

Comment:
Go guidance is for error messages to be uncapitalized and *not* end with a period.